### ISSUE TITLE
Program crashes when trying to export in fullscreen (macos)

### ISSUE THREAD

**Issue by AliahX** [kosua20/MIDIVisualizer#142]
MidiVisualizer crashes upon trying to export a video while in fullscreen.

Error log copied from the terminal.
```
[EXPORT]: Processing frame 1/14843.2023-08-06 13:52:55.835 MIDIVisualizer[63577:5320155] NSWindowStyleMaskFullScreen cleared on a window outside of a full screen transition. Called from (
	0   AppKit                              0x00007ff812138d0c __25-[NSWindow setStyleMask:]_block_invoke + 125
	1   AppKit                              0x00007ff812138c37 NSPerformVisuallyAtomicChange + 132
	2   AppKit                              0x00007ff812138b44 -[NSWindow setStyleMask:] + 170
	3   MIDIVisualizer                      0x0000000102886de8 _glfwPlatformSetWindowResizable + 72
	4   MIDIVisualizer                      0x0000000102876c0d _Z13performAction12SystemActionP10GLFWwindowRN3glm5tvec4IiLNS2_9precisionE0EEE + 61
	5   MIDIVisualizer                      0x000000010287762b main + 2235
	6   dyld                                0x000000020466d52e start + 462
	7   dyld                                0x0000000204668000 dyld + 0
)
2023-08-06 13:52:55.837 MIDIVisualizer[63577:5320155] *** Terminating app due to uncaught exception 'NSGenericException', reason: 'NSWindowStyleMaskFullScreen cleared on a window outside of a full screen transition. Called from (
	0   AppKit                              0x00007ff812138d0c __25-[NSWindow setStyleMask:]_block_invoke + 125
	1   AppKit                              0x00007ff812138c37 NSPerformVisuallyAtomicChange + 132
	2   AppKit                              0x00007ff812138b44 -[NSWindow setStyleMask:] + 170
	3   MIDIVisualizer                      0x0000000102886de8 _glfwPlatformSetWindowResizable + 72
	4   MIDIVisualizer                      0x0000000102876c0d _Z13performAction12SystemActionP10GLFWwindowRN3glm5tvec4IiLNS2_9precisionE0EEE + 61
	5   MIDIVisualizer                      0x000000010287762b main + 2235
	6   dyld                                0x000000020466d52e start + 462
	7   dyld                                0x0000000204668000 dyld + 0
)'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007ff80f7f5e9b __exceptionPreprocess + 242
	1   libobjc.A.dylib                     0x00007ff80f557e48 objc_exception_throw + 48
	2   CoreFoundation                      0x00007ff80f7f5d02 +[NSException exceptionWithName:reason:userInfo:] + 0
	3   AppKit                              0x00007ff812138d49 __25-[NSWindow setStyleMask:]_block_invoke + 186
	4   AppKit                              0x00007ff812138c37 NSPerformVisuallyAtomicChange + 132
	5   AppKit                              0x00007ff812138b44 -[NSWindow setStyleMask:] + 170
	6   MIDIVisualizer                      0x0000000102886de8 _glfwPlatformSetWindowResizable + 72
	7   MIDIVisualizer                      0x0000000102876c0d _Z13performAction12SystemActionP10GLFWwindowRN3glm5tvec4IiLNS2_9precisionE0EEE + 61
	8   MIDIVisualizer                      0x000000010287762b main + 2235
	9   dyld                                0x000000020466d52e start + 462
	10  dyld                                0x0000000204668000 dyld + 0
)
libc++abi: terminating with uncaught exception of type NSException
zsh: abort      /Applications/MIDIVisualizer.app/Contents/MacOS/MIDIVisualizer

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
```

- OS: macOS Monterey 12.5 (21G72)
- MIDIVisualizer version 7.0(1)
- GPU model Apple M2
- RAM available 8GB

Hope you're able to fix it sometime :)

**Comment by kosua20** [comment]
Hello, and thank you for reporting this issue! I've managed to reproduce the problem. The issue has been fixed in recent versions of the library I'm using for window/input management, so I've updated the version used by MIDIVisualizer. I'm planning on releasing a new version of MIDIVisualizer in the coming weeks, which will contain the fix.

**Comment by AliahX** [comment]
Thanks very much!

**Comment by kosua20** [comment]
Hello, this has been fixed in the latest [7.1 release](https://github.com/kosua20/MIDIVisualizer/releases/latest)!
